chapter2/tests/t5/chapter2-l2-t5-get-file-unique-extensions.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Script get a folder as an argument and return a list of unique extensions of all files (including in the nested directories)
+
+# check for the number of passed arguments
+if [[ $# -ne 1 ]]; then
+    echo "You must specify one folder"
+    exit 1
+fi
+
+folder=$1
+
+# search folder contents, output content without extension
+find "$folder" -type f | while read  file; do
+
+  # check files that contain only one dot in the name
+  if [[ $(echo "$file" | grep -o "\." | wc -l) -eq 1  ]]; then
+    if [[ ! "${file}" == *"/."* ]]; then
+      echo "${file/*./}"
+    fi
+
+  # check files that contain more than one dot in the name
+  elif [[ $(echo "$file" | grep -o "\." | wc -l) > 1 ]]; then
```

Comment:
```suggestion
  elif [[ $(echo "$file" | grep -o "\." | wc -l) -gt 1 ]]; then
```
https://www.shellcheck.net/wiki/SC2071

Reply:
done